README.md

```diff
@@ -1,3 +1,6 @@
+
+[![Build Status](https://secure.travis-ci.org/Enide/polyglot-maven-m2e.png)](http://travis-ci.org/Enide/polyglot-maven-m2e)
```

Comment:
sure, but that would mean that https://github.com/jbosstools/m2e-polyglot-poc would stay as main

Could you comment in https://github.com/jbosstools/m2e-polyglot-poc/issues/8